config/testdata/conf.slack-both-file-and-url.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  slack_api_url: "http://mysecret.example.com/"
+  slack_api_url_file: '/global_file'
+
+route:
+  receiver: 'slack-notifications'
+  group_by: [alertname, datacenter, app]
+
+receivers:
+- name: 'slack-notifications'
+  slack_configs:
+  - channel: '#alerts1'
+    text: 'test'
+    
```

Comment:
No newline at end of file. (But some whitespace on line 22, which just needs to be deleted?)